tests/Browser/AdminListDailyMenuTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Facebook\WebDriver\Chrome\ChromeOptions;
+use \App\DailyMenu;
+use \App\Food;
+use \App\Category;
+
+
+class AdminListDailyMenuTest extends DuskTestCase
+{
+    use DatabaseTransactions;
+    use WithoutMiddleware;
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test URL admin view list daily menu.
+     *
+     * @return void
+     */
+    public function testURL()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+            ->visit('/dashboard')
+            ->clickLink('Management')
+            ->clickLink('Daily Menu Management')
+            ->assertPathIs('/daily-menus');
+        });
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test show content.
+     *
+     * @return void
+     */
+    public function testContent()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+                ->visit('/daily-menus')
+                ->assertSee("LIST DAILY MENUS");
+        });
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test count object in list=10.
+     *
+     * @return void
+     */
+    public function testPaginationWith10Rows()
+    {
+        $category = factory(Category::class, 1)->create();
+        $foods = factory(Food::class, 10)->create(['category_id' => 1]);
```

Comment:
Use `factory(Food::class, 10)` is not good. You create 10 record -> count -> compare with 10. What happen when limit per page changed?
For example:
`ITEMS_PER_PAGE` in model `Food` changed to 5. So on one page, we have 5 records. You create 10 records but in list display only 5. Your test fails.